src/components/meals/Meals.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import styles from '../../styles';
+import {
+  MealCategories,
+  MealInput,
+  MealList,
+  MealsComponentProps,
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  FlatList,
+} from '../../models/meals.model';
+
+const MAX_ITEMS = 3;
```

Comment:
perhaps a more explicit name? a max of what items?

Reply:
explicit - indicative